desk/tests/app/groups.hoon

```diff
@@ -0,0 +1,32 @@
+/-  g=groups
+/+  *test-agent
+/=  groups-agent  /app/groups
+|%
+++  dap  %groups-test
+++  the-wire  /groups/(scot %p ~zod)/test/updates
+++  the-path  (weld the-wire /init)
+++  the-dock  [~zod dap]
+++  the-group  [%test 'test' '' '' '' [%open ~ ~] ~ |]
+++  retry  (weld /~/retry the-wire)
+++  test-subscription-loop
+  %-  eval-mare
+  =/  m  (mare ,~)
+  ::  init and add group
+  ;<  *  bind:m  (do-init %groups-test groups-agent)
+  ;<  *  bind:m  (jab-bowl |=(b=bowl b(our ~dev, src ~dev)))
+  ;<  *  bind:m  (do-poke %group-join !>([[~zod %test] &]))
+  ;<  *  bind:m  (jab-bowl |=(b=bowl b(our ~dev, src ~zod)))
```

Comment:
See also `+set-src`, which may be more appropriate here.

(We don't have a `+set-our` yet, but I'd probably set that explicitly before `+do-init`, to make sure it's the same during the entire lifecycle.)